CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Version 0.2.5
```

Comment:
I didn't notice this earlier, but this should be `## [Unreleased]`, since the next version isn't necessarily 0.2.5. Other than that, everything's looking good now.

Reply:
No worries, have that updated now.